templates/base-devguide.html

```diff
@@ -12,6 +12,7 @@
 <head>
 <title>{% block title %}{% endblock %} - Polymer Project</title>
 {% include 'templates/head-meta.html' %}
+<style is="custom-style" include="iron-doc-default-theme"></style>
```

Comment:
When navigating from a different page on the docs site, the head is not copied, so these styles won't be available. Also with the upcoming changes in #2207, Polymer (and therefore `custom-style` and `<dom-module id="iron-doc-default-theme">`) will be async loaded, so including these in `<head>` is not appropriate. I moved these to iron-doc-viewer.html in commit https://github.com/Polymer/docs/pull/2232/commits/331c84719e3845310aadba2fdabf57edf10e1da2 (in PR #2207).